askama_axum/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "askama_axum"
-version = "0.10.0"
+version = "0.1.0"
```

Comment:
Um, I guess you meant to hit the "1" key one more often

Reply:
I actually didn't... This is a new package that hasn't been published before, so we can just start from 0.1.0.